zircon.core/common/src/main/kotlin/org/hexworks/zircon/internal/component/impl/DefaultSlider.kt

```diff
@@ -0,0 +1,254 @@
+package org.hexworks.zircon.internal.component.impl
```

Comment:
I've looked several times but I can't see where the left/right arrow rendering code is.

Reply:
It is only in the example. Let's keep it as it is and discuss how we would add the buttons in a later PR.